src/components/pages/LandingPrograms/CoderYoga.js

```diff
@@ -26,13 +26,15 @@ export default function CoderYoga() {
           </div>
           <div className="pl__na__right">
             <div className="pl__na__right__container">
-              <Title className="pl__na__coder__yoga__title">CODERYOGA</Title>
-              <Text className="pl__na__right__info__container">
+              <Title className="pl__na__coder__yoga__title" level={2}>
+                CODER-YOGA
```

Comment:
Is Coder-Yoga presented this way in our docs, or should it be 2 separate words? 🤔

Reply:
@dstrazzeri mentioned in a meeting just now that our docs refer to it as "CoderYoga" - maybe we should make this change before merging this code up!